services/movie_session.py

```diff
@@ -42,3 +44,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    row_seats = Ticket.objects.filter(
+        movie_session_id=movie_session_id
+    ).values("row", "seat")
+
+    return list(row_seats)
```

Comment:
no need to store row_seats in a variable - just return whole expression in list()